packages/core/admin/admin/src/pages/SettingsPage/pages/Users/EditPage/utils/api.js

```diff
@@ -1,12 +1,5 @@
 import { getFetchClient } from '@strapi/helper-plugin';
 
-const fetchUser = async (id) => {
-  const { get } = getFetchClient();
-  const { data } = await get(`/admin/users/${id}`);
-
-  return data.data;
-};
-
 const putUser = async (id, body) => {
```

Comment:
Would we add these mutations to the `useAdminUsers`?

Reply:
Why not. Could be a good follow-up :)